azure-pipelines.windows.yml

```diff
@@ -0,0 +1,178 @@
+name: Roaster Windows Build
+
+trigger:
+- master
+
+pool:
+  name: $(AGENT_POOL_NAME)
+
+stages:
+- stage: prep
+  jobs:
+    - job: vsbuildtools
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+        parameters:
+          stage_name: vsbuildtools
+
+    - job: cmake
+      dependsOn: vsbuildtools
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: curl
+      dependsOn: cmake
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: cuda
+      dependsOn: curl
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: cudnn
+      dependsOn: cuda
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: tensorrt
+      dependsOn: cudnn
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: intel
+      dependsOn: tensorrt
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+    - job: mklml
+      dependsOn: intel
+      steps:
+      - template: azure-pipelines/build_stage.windows.yml
+
+- stage: build
+  dependsOn: prep
+  jobs:
+    - job: zlib
```

Comment:
Recommend to group pkgs like those in `docker/centos/{infra,util,misc,dl,ort}`, to reduce complexity of `dependsOn` and make it easier to new pkgs.
Your template should be able to handle multiple inputs.

Reply:
Good suggestion. Fixed.